cluster-up/cluster/external/provider.sh

```diff
@@ -20,6 +20,7 @@ docker_tag=\${DOCKER_TAG}
 docker_prefix=\${DOCKER_PREFIX}
 manifest_docker_prefix=\${DOCKER_PREFIX}
 image_pull_policy=\${IMAGE_PULL_POLICY:-Always}
+kubectl=\${KUBECTL}
```

Comment:
What happens if $KUBECTL is not set in the env? should we default back to assuming that it is installed and can be found under $PATH ?

Similar to how it is handled for oc below
https://github.com/kubevirt/kubevirtci/pull/1238/files#diff-4f9667aaacd597e4d77e9ebe5cf91003e261649e70bce7b591f98df8104beee5L25

Reply:
Hi @brianmcarey 
I haven't tried that let me check that once.

Reply:
Yes @brianmcarey  It is working, We can default back to assuming that it is installed and can be available in the PATH, similar to oc

Reply:
Updated the pr with the changes.